### PR TITLE
Networking V2: add QoS Policy resource

### PR DESCRIPTION
**Pull request by ozerovandrei**
Implement "openstack_networking_qos_policy_v2" resource with acceptance
tests and website documentation.

For #760

### CODE REVIEW

File: openstack/types.go [hunk 1]
```diff
@@ -306,3 +307,9 @@ type SiteConnectionCreateOpts struct {
 	siteconnections.CreateOpts
 	ValueSpecs map[string]string `json:"value_specs,omitempty"`
 }
+
+// QoSPolicyCreateOpts represents the attributes used when creating a new QoS policy.
+type QoSPolicyCreateOpts struct {
```
Comment (jtopjian) [inline openstack/types.go]: nit: add this to `openstack/networking_qos_policy_v2.go`. The remaining types that are in `types.go` are resources that still need to be cleaned up (I think there are a few networking ones in queue? The VPN ones are untouched).

Comment (ozerovandrei) [reply openstack/types.go]: Not really. Some of them were already refactored.
But I will have this in mind and those types will be migrated.

**Comment by ozerovandrei** [comment]
recheck

**Comment by ozerovandrei** [comment]
recheck

**Comment by jtopjian** [comment]
recheck

**Pull request merged**